src/parsers/manifestjson.js

```diff
@@ -121,8 +121,22 @@ export default class ManifestJSONParser extends JSONParser {
         }
       }
     }
+
     if (support.firefox_android) {
-      const versionAddedAndroid = support.firefox_android.version_added;
+      let versionAddedAndroid;
+      if (Array.isArray(support.firefox_android)) {
+        // We want to retrieve the last `version_added` value (i.e. the highest
+        // browser version).
```

Comment:
@willdurand I think that for this case ([1]) the approach in this patch may be ok concerning, but I was trying to think to other possible cases and if this approach will always be ok, and so I was wondering: did you checked if there are any other array entries in the rest of the metadata from this repository?

I guess that there are not many webextension using specific values on both min and max versions supported, my guess would be that most of the webextensions are more likely just setting the minVersion and not both), and so for extensions that do only specify the minVersion this simple check seems to be good enough.

[1]: I'm pretty sure that the gap in the supported versions is due to the switch to Fenix, and the most recent entry is the first version where the API was supported again.